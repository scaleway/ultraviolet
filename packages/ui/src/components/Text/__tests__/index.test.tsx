import { shouldMatchEmotionSnapshot } from '@utils/test'
import { describe, test } from 'vitest'
import { Text, textVariants } from '..'

describe('Text', () => {
  test.each(textVariants)('renders correctly with type="%s"', variant =>
    shouldMatchEmotionSnapshot(
      <Text as="div" variant={variant}>
        {variant}
      </Text>,
    ),
  )

  test(`renders correctly with tooltip`, () =>
    shouldMatchEmotionSnapshot(
      <div style={{ marginBottom: 16, marginTop: 8, width: 500 }}>
        <Text as="div" oneLine variant="body">
          This text is quite long. Lorem ipsum dolor sit amet, consectetur
          adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
          magna aliqua.
        </Text>
      </div>,
    ))

  test(`renders correctly with placement`, () =>
    shouldMatchEmotionSnapshot(
      <div style={{ marginBottom: 16, marginTop: 8, width: 500 }}>
        <Text as="div" placement="end" variant="body">
          This text is quite long. Lorem ipsum dolor sit amet, consectetur
          adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
          magna aliqua.
        </Text>
      </div>,
    ))

  test(`renders correctly with dir`, () =>
    shouldMatchEmotionSnapshot(
      <div style={{ marginBottom: 16, marginTop: 8, width: 500 }}>
        <Text as="div" dir="rtl" oneLine variant="body">
          This text is quite long. Lorem ipsum dolor sit amet, consectetur
          adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
          magna aliqua.
        </Text>
      </div>,
    ))

  test(`renders correctly with htmlFor`, () =>
    shouldMatchEmotionSnapshot(
      <div style={{ marginBottom: 16, marginTop: 8, width: 500 }}>
        <Text as="div" htmlFor="test" variant="body">
          This text is quite long. Lorem ipsum dolor sit amet, consectetur
          adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
          magna aliqua.
        </Text>
      </div>,
    ))

  test(`with multiple nested children renders correctly`, () =>
    shouldMatchEmotionSnapshot(
      <Text as="div" variant="body">
        Lorem
        <span>Ipsum</span>
        <Text as="span" variant="heading">
          <span>Dolor</span>
        </Text>
        Sit
      </Text>,
    ))

  test(`with prominence stronger on non neutral`, () =>
    shouldMatchEmotionSnapshot(
      <Text as="div" prominence="stronger" sentiment="danger" variant="body">
        Lorem Ipsum
      </Text>,
    ))
  test(`with italic`, () =>
    shouldMatchEmotionSnapshot(
      <Text as="div" italic variant="body">
        Lorem Ipsum
      </Text>,
    ))
  test(`with underline`, () =>
    shouldMatchEmotionSnapshot(
      <Text as="div" underline variant="body">
        Lorem Ipsum
      </Text>,
    ))

  test(`with disabled`, () =>
    shouldMatchEmotionSnapshot(
      <Text as="div" disabled variant="body">
        Lorem Ipsum
      </Text>,
    ))

  test(`with monochrome`, () =>
    shouldMatchEmotionSnapshot(
      <Text as="div" sentiment="black" variant="body">
        Lorem Ipsum
      </Text>,
    ))

  test(`renders correctly with whiteSpace`, () =>
    shouldMatchEmotionSnapshot(
      <Text as="div" variant="body" whiteSpace="nowrap">
        This text is quite long. Lorem ipsum dolor sit amet, consectetur
        adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
        magna aliqua.
      </Text>,
    ))
})
