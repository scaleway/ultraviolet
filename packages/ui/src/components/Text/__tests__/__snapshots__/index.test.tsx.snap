// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Text > renders correctly with dir 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      style="margin-bottom: 16px; margin-top: 8px; width: 500px;"
    >
      <div
        class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_true__m4c9ow7 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
        dir="rtl"
      >
        This text is quite long. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with htmlFor 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      style="margin-bottom: 16px; margin-top: 8px; width: 500px;"
    >
      <div
        class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
        for="test"
      >
        This text is quite long. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with placement 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      style="margin-bottom: 16px; margin-top: 8px; width: 500px;"
    >
      <div
        class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
        style="--qabug40: end;"
      >
        This text is quite long. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with tooltip 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      style="margin-bottom: 16px; margin-top: 8px; width: 500px;"
    >
      <div
        class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_true__m4c9ow7 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
      >
        This text is quite long. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="body" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      body
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="bodySmall" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_bodySmall__m4c9own style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      bodySmall
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="bodySmallStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_bodySmallStrong__m4c9owo style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      bodySmallStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="bodySmallStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_bodySmallStronger__m4c9owp style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      bodySmallStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="bodyStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_bodyStrong__m4c9owq style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      bodyStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="bodyStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_bodyStronger__m4c9owr style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      bodyStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="caption" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_caption__m4c9ows style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      caption
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="captionSmall" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_captionSmall__m4c9owt style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      captionSmall
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="captionSmallStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_captionSmallStrong__m4c9owu style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      captionSmallStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="captionSmallStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_captionSmallStronger__m4c9owv style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      captionSmallStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="captionStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_captionStrong__m4c9oww style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      captionStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="captionStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_captionStronger__m4c9owx style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      captionStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="code" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_code__m4c9owy style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      code
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="codeStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_codeStrong__m4c9owz style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      codeStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="codeStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_codeStronger__m4c9ow10 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      codeStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="heading" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_heading__m4c9ow11 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      heading
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingLarge" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingLarge__m4c9ow12 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingLarge
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingLargeStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingLargeStrong__m4c9ow13 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingLargeStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingLargeStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingLargeStronger__m4c9ow14 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingLargeStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingSmall" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingSmall__m4c9ow15 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingSmall
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingSmallStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingSmallStrong__m4c9ow16 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingSmallStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingSmallStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingSmallStronger__m4c9ow17 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingSmallStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingStrong" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingStrong__m4c9ow18 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingStrong
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with type="headingStronger" 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_headingStronger__m4c9ow19 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      headingStronger
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > renders correctly with whiteSpace 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
      style="--qabug41: nowrap;"
    >
      This text is quite long. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > with disabled 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_true__m4c9ow1a"
    >
      Lorem Ipsum
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > with italic 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_true__m4c9ow3 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      Lorem Ipsum
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > with monochrome 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_sentiment_black__m4c9owg style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_28__m4c9ow24 style_undefined_compound_72__m4c9ow3c"
    >
      Lorem Ipsum
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > with multiple nested children renders correctly 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      Lorem
      <span>
        Ipsum
      </span>
      <span
        class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_heading__m4c9ow11 style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
      >
        <span>
          Dolor
        </span>
      </span>
      Sit
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > with prominence stronger on non neutral 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_sentiment_danger__m4c9owd style_prominence_stronger__m4c9owk style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_10__m4c9ow1m style_undefined_compound_74__m4c9ow3e"
    >
      Lorem Ipsum
    </div>
  </div>
</DocumentFragment>
`;

exports[`Text > with underline 1`] = `
<DocumentFragment>
  <div
    data-testid="testing"
  >
    <div
      class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_true__m4c9ow5 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_body__m4c9owm style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
    >
      Lorem Ipsum
    </div>
  </div>
</DocumentFragment>
`;
